notebooks/Linear Regression.ipynb

```diff
@@ -0,0 +1,1288 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x1255224e0>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "y_train_predict=model.predict(X_train)\n",
+    "y_test_predict=model.predict(X_test)\n",
+    "#print(y_train_predict[:,0])\n",
+    "y_train_plot=y_train.as_matrix()\n",
+    "y_test_plot=y_test.as_matrix()\n",
+    "#y_train_predict_plot=y_train_predict.as_matrix()\n",
+    "\n",
+    "plt.scatter(y_train_plot[:,0],y_train_predict[:,0])\n",
+    "plt.xlabel(\"observed\")\n",
+    "plt.ylabel(\"predicted\")\n",
+    "plt.title(\"Predicted Ca\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### However, the model doesn't generalize well to the test data:\n",
+    "\n",
+    "If we plot the observed vs. predicted values for each sample in the test set, we can see that a lot of the variance isn't explained by the linear model.\n"
```

Comment:
I'd note the connection between "variance explained" and "$R^2$" here, so folks make the connection. I'd also, as above, frame this in terms of predictions as well.